src/wifi-menu

```diff
@@ -242,13 +242,18 @@ cd /  # We do not want to spawn anything that can block unmounting
 
 INTERFACE=$1
 if [[ -z "$INTERFACE" ]]; then
-    INTERFACE=(/sys/class/net/*/wireless/)
-    if [[ ${#INTERFACE[@]} -ne 1 || ! -d "$INTERFACE" ]]; then
-        report_error "Invalid interface specification"
+    INTERFACES=(/sys/class/net/*/wireless/)
+    if [[ ${#INTERFACES[@]} -gt 1 ]]; then
+        report_debug "Multiple wireless interfaces found: '${INTERFACES[@]}'"
+        INTERFACE_LIST=$(for IFACE in ${INTERFACES[@]}; do echo ${IFACE:15:-10}; done)
+        INTERFACE=$(dialog --no-items --menu "Select wireless interface you wish to use:" 13 50 12 $INTERFACE_LIST --stdout)
```

Comment:
Let's reuse the `INTERFACES` global variable and not introduce yet another one. I don't see much use for the debugging statement.

An article was missing in the message, and when adding it, it was rather long. What do you think of simply "Select the interface you wish to use". This is consistent with the message in the next screen ("Select the network you wish to use") and "wireless" is probably pretty redundant given the name of the tool.

I am curious why you chose a height of 13 lines. It's fine with me, but if there is no particular reason, then let's use the same dimensions as the next screen.

```suggestion
        INTERFACES=$(for INTERFACE in ${INTERFACES[@]}; do
            echo ${INTERFACE:15:-10}
        done)
        INTERFACE=$(dialog --no-items --menu "Select the interface you wish to use" 24 50 12 $INTERFACES --stdout)
```

Reply:
I just chose something smaller since 99% of the time there's 2 or fewer wireless interfaces, so it looked weird with all the unused space. I'm fine with maintaining the same dimensions.